frontend/src/app/components/containers/complaints/complaints.tsx

```diff
@@ -154,8 +146,22 @@ export const Complaints: FC<Props> = ({ defaultComplaintType }) => {
 export const ComplaintsWrapper: FC<Props> = ({ defaultComplaintType }) => {
   const defaultZone = useAppSelector(selectDefaultZone, shallowEqual);
   const currentOfficer = useAppSelector(selectCurrentOfficer(), shallowEqual);
+  const filters = getFilters(currentOfficer, defaultZone);
 
+  return (
+    <>
+      {filters && (
+        <ComplaintFilterProvider {...filters}>
+          <Complaints defaultComplaintType={defaultComplaintType} />
+        </ComplaintFilterProvider>
+      )}
+    </>
+  );
+};
+
+export const getFilters = (currentOfficer: any, defaultZone: any) => {
```

Comment:
the getFilters function shouldn't be exported, and you'll want to move it to the `ComplaintsWrapper`